helpers.go

```diff
@@ -932,6 +938,16 @@ func verifyRuleResult(
 	return nil
 }
 
+// Will exclude the rule from counting if excludedString has values
+func isExcluded(exclude interface{}) bool {
+	if excludedString, ok := exclude.(string); ok {
+		if excludedString != "" {
```

Comment:
are we returning true on any value to support true/TRUE/yes/YES/on/ON? Was wondering what would happen if someone says "exclude_from_count: FALSE" in the content..

Reply:
Let's change it to `Yes`